hw10/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homework-10</title>
+  </head>
+
+  <body>
+    <main>
+      <h1 class="header">Blog</h1>
+      <div class="text">
+        <p class="text__basic">
+          Dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+          Duis aute irure dolor in reprehenderit in voluptate velit.
+        </p>
+      </div>
+      <div class="gallery">
+        <div class="gallery-left">
+          <img class="gallery_img" src="img/kiwi.jpg" alt="" />
+          <time class="gallery__data" datetime="2023-11-20">20 NOV</time>
+          <div class="gallery__gradient"></div>
+          <a href="#">
+            <h4 class="gallery__headline">
+              Green Smoothies: Too Much of a Good Thing?
+            </h4>
+          </a>

```

Comment:
I see the link - but it doesn`t cover all card, so link will not work on all cart. Let`s fix it

Also you add hover effect to the img - but it doesn`t work when you hover texts. Let`s fix it also